### DISCUSSION TITLE
How I could create a heatmap using the objects API?

### DISCUSSION THREAD

**Discussion by aleixalcacer** [mwaskom/seaborn#3285] [Q&A] [answered]
Hi,

I'm looking for a function similar to `geom_tile()` (`ggplot2` package) to be able to create heatmaps with the objects API. Is there any alternative to create a heatmap using the object API?

If not, is it planned to implement it?

Thanks,
Aleix

**Comment by thuiop** [comment, marked as answer]
For now, you can't. It is planned to support 2d histogram in the future though, which would possibly mean heatmap too.

**Reply by aleixalcacer** [reply]
Thank you :)

**Reply by yelbouhassani** [reply]
Any update on this feature? It would be great if it were made possible 🙏

**Reply by thuiop** [reply]
The development of seaborn has been stalled for a while, so unfortunately no.
